.github/workflows/action.yml

```diff
@@ -0,0 +1,41 @@
+name: Combine members data 
+on : 
+    push : 
+        branches:
+            - main
+jobs: 
+    job1:
+        name : data copy into dist
```

Comment:
Let's try and be consistent please. I see later that we are using title-case, but here everything is lower-cased